src/stores/store.ts

```diff
@@ -1,15 +1,17 @@
+import type { Utxo } from 'marina-provider';
 import { writable } from 'svelte/store';
 
-
 export interface MarinaStore {
   installed: boolean;
   enabled: boolean;
+  utxos: Utxo[];
 }
 
-const initialState = { installed: false, enabled: false, network:"liquid" }
+const initialState = { installed: false, enabled: false, network: "liquid", utxos: [] };
```

Comment:
I would keep utxos in a different store (potentially we could use `svelte-marina-button` which handles this marinaStore as a cache for connection status only

Reply:
done